### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Cliffmax85**

### CODE REVIEW

File: src/App.test.jsx [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+describe('Header', () => {
+test('Should render the header', async () => {
+    render(
+          <App />
+    )
+    
+    screen.getByAltText('Alchemy Logo');
+    // console.log('||||||||ALT||||||||', altText);
+
+    screen.debug();
```
Comment (BenFrancka) [inline src/App.test.jsx:14]: these `screen.debug()`'s are kind of like `console.log()`'s.  They are useful for troubleshooting/validation but you can remove them after your test is written out and passing.

File: src/views/Home/Home.test.jsx [hunk 2]
```diff
@@ -11,5 +14,47 @@ const user = {
...
+  // NAME
+  screen.getByText(/vonta/i);
+  // Motto
+  screen.getByText(/res non verba/i);
+  // Interests
+  screen.getByText(/interests/i);
+  // Avatar
+  screen.getByAltText(/avatar/i);
+  // Header
+  screen.getByAltText(/header/i);
```
Comment (BenFrancka) [inline src/views/Home/Home.test.jsx:32]: I would set all of these elements you are finding on the page to a variable, and then you can `expect(variableName).toBeInTheDocument()` or a myriad of other `expect` statements.  These make your test more readable by explicitly stating what you are evaluating for.

File: src/views/Home/Home.test.jsx [hunk 3]
```diff
@@ -11,5 +14,47 @@ const user = {
...
+  // List of interests
+  const likesArr = [];
```
Comment (BenFrancka) [inline src/views/Home/Home.test.jsx:34]: This is a clever workaround!  Remember to read through the react docs and look at all the different roles you can find by like `list`.  You can grab the list directly and test it's length and contents like any array.

File: src/views/Home/Home.test.jsx (hunk 2 above)
Comment (Cliffmax85) [reply src/views/Home/Home.test.jsx:32]: That makes sense. I'll make sure to do this in the from scratch.

**Pull request merged**